Orange/canvas/document/usagestatistics.py

```diff
@@ -105,6 +105,31 @@ def log_node_added(self, widget_name, extended_widget=None):
     def set_node_type(self, addition_type):
         self.__node_addition_type = addition_type
 
+    def send_statistics(self):
+        def store_data(d):
```

Comment:
We probably do not need to store data in this function - as far as I understand we only need to clear when it was sent successfully.

Reply:
You are right, I will remove this part.